tests/tests.go

```diff
@@ -209,6 +210,26 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() (time.Duration, error) {
+	managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(managerNamespace).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred(), "Should successfully get Deployment object")
+	Expect(len(managerDeployment.Spec.Template.Spec.Containers) != 0).To(BeTrue(), "Container array size should be greater then zero")
+
+	for _, container := range managerDeployment.Spec.Template.Spec.Containers {
+		if container.Name == deploymentContainerName {
+			for _, arg := range container.Args {
+				waitTime := 0
+				if elementsParsed, err := fmt.Sscanf(arg, "--wait-time=%d", &waitTime); elementsParsed == 1 {
```

Comment:
I think this function shouldn't handle this error with `Expect` and just return it `return 0, error`
so the caller function will return an error like: 
"Should successfully get wait-time argument: <parse error from `Sscanf`>"

Reply:
I agree